packages/ui-components/src/components/DecisionTree/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+export * from './DecisionTree';
```

Comment:
Another name I considered is `HorizontalTree`

Reply:
I think I actually prefer `HorizontalTree`. `DecisionTree` feels as though it implies some kind of meaning about the data within the tree (ie. it's used for making decisions, eg. 'if I perform action A, event B will happen')

Reply:
Happy to rename it!